vue-frontend/src/components/contribution/GithubContribution.vue

```diff
@@ -0,0 +1,666 @@
+
+          link: "https://github.com/canopas/web-developer-roadmap-2023",
+          language: "JAVA",
+          color: "tw-bg-[#F1E05A]",
+          forks: "10",
+          title: "Web Development Roadmap",
+          description:
+            "Web Developer Roadmap is a path to understand web development including...",
+          stars: "72",
+          contributors: [
+            {
+              image: contributor8,
+            },
+            {
+              image: contributor10,
+            },
+          ],
+        },
+        {
+          id: 9,
+          image: [web3400w, web3800w],
+          deskImage: [deskweb3400w, deskweb3800w],
+
+          link: "https://github.com/canopas/canopas-blog",
+          language: "JavaScript",
+          color: "tw-bg-[#F1E05A]",
+          forks: "00",
+          title: "Canopas Blog website",
+          description:
+            "We intend to keep this open source. Plan is to keep the repository up to date with...",
+          stars: "02",
+          contributors: [
+            {
+              image: contributor8,
+            },
+            {
+              image: contributor11,
+            },
+            {
+              image: contributor12,
+            },
+            {
+              image: contributor13,
+            },
+          ],
+        },
+        {
+          id: 10,
+          image: [web4400w, web4800w],
+          deskImage: [deskweb4400w, deskweb4800w],
+          link: "https://github.com/canopas/canopas-website",
+          language: "Vue",
+          color: "tw-bg-[#41B883]",
+          forks: "02",
+          title: "Canopas Website",
+          description:
+            "We intend to keep this open source. Plan is to keep the repository up to date with latest...",
+          stars: "07",
+          contributors: [
+            {
+              image: contributor8,
+            },
+            {
+              image: contributor1,
+            },
+            {
+              image: contributor12,
+            },
+            {
+              image: contributor14,
+            },
+            {
+              image: contributor15,
+            },
+            {
+              image: contributor10,
+            },
+            {
+              image: contributor11,
+            },
+            {
+              image: contributor13,
+            },
+          ],
+        },
+      ],
+    };
+  },
+  components: {
+    Swiper,
+    SwiperSlide,
+    FontAwesomeIcon,
+  },
+  mounted() {
+    this.contributions = this.allContributions.slice(0, 3);
+  },
+
+  methods: {
+    openContribution(contribution) {
+      window.open(contribution.link, "_blank");
+    },
+    onSlideChange(index) {
+      if (index <= 1) {
+        this.activeIndex = 0;
+      } else if (index > 1 && index <= 4) {
+        this.activeIndex = 1;
+      } else if (index > 3 && index <= 9) {
+        this.activeIndex = 2;
+      }
+    },
+    playSwiper(play) {
```

Comment:
Is it needed? As per the changes i see, it's just used for assigning boolean values.
 `@touchstart.passive="playSwiper(false)"` can be replaced with 
`@touchstart.passive="false"`